Gemfile

```diff
@@ -5,4 +5,4 @@ gem 'jekyll-sitemap'
 gem 'jekyll-redirect-from'
 gem 'jekyll-seo-tag'
 gem 'jekyll-coffeescript'
-gem 'github-pages'
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
Does this include the remote theme? I didn’t realize it did.

Reply:
according to the docs, all this line does is 

"Note: As of Jekyll 3.2, the default Jekyll site contains a Gemfile that locks Jekyll to the Gem version you build it with. To instead lock it to the version used by GitHub Pages, you'll uncomment the gem "github-pages", group: :jekyll_plugins line."